l3build-check.lua

```diff
@@ -595,7 +595,9 @@ function runcheck(name, hide)
     return 1
   end
   local checkengines = checkengines
-  if options["engine"] then
+  if options["stdengine"] then
+    checkengines = {stdengine}
+  elseif options["engine"] then
     checkengines = options["engine"]
```

Comment:
not really for this pr but seeing this line: shouldn't in case of -e the checkengines variable set to the \cap of "engine" and the existing "checkengines" of a config file?

Reply:
Not necessarily - you can add `-f` for force running with an 'unsuitable' engine

Reply:
```lua
-- Sanity check
function check_engines(config)
  if options["engine"] and not options["force"] then
    -- Make a lookup table
    local t = { }
    for _, engine in pairs(checkengines) do
      t[engine] = true
    end
    for _, engine in pairs(options["engine"]) do
      if not t[engine] then
        print("\n! Error: Engine \"" .. engine .. "\" not set up for testing with configuration \"" .. config .. "\"!")
        print("\n  Valid values are:")
        for _, engine in ipairs(checkengines) do
          print("  - " .. engine)
        end
        print("")
        exit(1)
      end
    end
  end
end
```

Reply:
I nevery really understood the use case for force :-) but yes, that makes it a little more complicated. But in case of asking for `-e`without `-f`: wouldn't it be more oportune to simply not do any tests for any of the "e" engines that are not in "checkengines" of a particular config rather calling it an error? Well maybe not, dunno

Reply:
Well we could drop `--force`: it was there for some setup stuff which likely we no longer need (and I think `-s` makes redundant).

Reply:
That could be done separate from the PR of course

Reply:
we could I guess, and yes it should be done in a separate pr.